### ISSUE TITLE
Type Error when specifying initialData (and other options)

### ISSUE THREAD

**Issue by thomasballinger** [get-convex/convex-react-query#2]
Just adding `initialData: []` causes a monster error in the example in this repo

```
function App() {
  const { data, error, isPending } = useQuery({
    // This query updates reactively.
    ...convexQuery(api.messages.list, {}),
    initialData: [],
  });
```
![image](https://github.com/user-attachments/assets/5f962055-419d-44be-bfe5-98b221e0fca7)

**Comment by thomasballinger** [comment]
Looks like the old method-based way still works. I think the standalone function can work, we've just typed it wrong.
![image](https://github.com/user-attachments/assets/031e98f6-c88f-485c-9cda-bce6e63497cc)

**Comment by thomasballinger** [comment]
fixed in 0.0.0-alpha.4
